lib/ffmpegCommandService.js

```diff
@@ -515,28 +519,32 @@ class FfmpegCommandService {
         return {filter, maps};
     }
 
-    _createFilterForTextOverlay(textOverlay, i) {
+    _createFilterForTextOverlay(textOverlay, i, command) {
+        // TODO: doesn't appear to work?
         const bashCharsToEscapeRegex = /!|'|"/g;
 
         // @TODO: add handling for escaping special characters in bash.  Know it's at least ! that needs escaping.
         const text = textOverlay.text.replace(bashCharsToEscapeRegex, "\\$&");
 
-        const defaultMargin = this._fontConfig.defaultMargin || DEFAULT_FONT_MARGIN;
+        // scale the margin with the same ratio as font size
+        let defaultMargin = this._fontConfig.defaultMargin || DEFAULT_FONT_MARGIN;
+        defaultMargin = (16 / defaultMargin) * (command.output.dimensions.height / 16);
 
         const x = this._calculateHorizontalPosition(textOverlay.xLoc, defaultMargin);
         const y = this._calculateVerticalPosition(textOverlay.yLoc, defaultMargin);
+        const size = (16 / textOverlay.fontSize) * (command.output.dimensions.height / 16);
 
         const fontFile = Path.resolve(this._fontConfig.fontsDirectory, this._fontLookupObj[textOverlay.fontName].fontFilePath);
         let filter = ` drawtext=enable=1:text='${text}':x=${x}:y=${y}`;
-        filter += `:fontfile=${fontFile}:fontsize=${textOverlay.fontSize}`;
+        filter += `:fontfile=${fontFile}:fontsize=${size}`;
 
         // See example near bottom of this page for the copy / pasted formula:  https://ffmpeg.org/ffmpeg-filters.html#Examples-51
         // fontcolor_expr=ff0000%{eif\\\\: clip(255*(1*between(t\\, $DS + $FID\\, $DE - $FOD) + ((t - $DS)/$FID)*between(t\\, $DS\\, $DS + $FID) + (-(t - $DE)/$FOD)*between(t\\, $DE - $FOD\\, $DE) )\\, 0\\, 255) \\\\: x\\\\: 2 }
         const ds = textOverlay.fadeIn.startTime; // display start
         const de = textOverlay.fadeOut ? textOverlay.fadeOut.startTime + textOverlay.fadeOut.duration : 1000000; // display end; 1000000 is BIG in seconds and should result in no fade out
         const fid = textOverlay.fadeIn.duration; // fade in duration
         const fod = textOverlay.fadeOut ? textOverlay.fadeOut.duration : 1; // fade out duration; defaulting to 1 to avoid divide by 0 error
-        filter += `:fontcolor_expr=${textOverlay.fontColor}%{eif\\\\\\\\: clip(255*${textOverlay.fontAlpha}*(1*between(t\\\\, ${ds + fid}\\\\, ${de - fod}) + ((t - ${ds})/${fid})*between(t\\\\, ${ds}\\\\, ${ds + fid}) + (-(t - ${de})/${fod})*between(t\\\\, ${de - fod}\\\\, ${de}) )\\\\, 0\\\\, 255) \\\\\\\\: x\\\\\\\\: 2 }`;
```

Comment:
Not sure I need this. Was testing whether I could do text with no fade, but it doesn't like 0...